ddtrace/util.py

```diff
@@ -1,17 +1,17 @@
 # [Backward compatibility]: keep importing modules functions
 from .internal.utils.deprecation import deprecated
-from .internal.utils.deprecation import deprecation
 from .internal.utils.formats import asbool
 from .internal.utils.formats import deep_getattr
 from .internal.utils.formats import get_env
 from .internal.utils.wrappers import safe_patch
 from .internal.utils.wrappers import unwrap
+from .vendor.debtcollector.removals import removed_module
 
 
-deprecation(
-    name="ddtrace.util",
-    message="Use `ddtrace.utils` package instead",
-    version="1.0.0",
+removed_module(
+    module="ddtrace.util",
+    replacement="ddtrace.utils",
```

Comment:
but we are deprecating `ddtrace.utils` as well right? so we should just say this is going away?